lib/LibXML/Node.rakumod

```diff
@@ -155,12 +163,13 @@ submethod DESTROY {
     self.raw.Unreference;
 }
 
-multi method box(anyNode:D $_) {
-    .Reference;
-    my $class := box-class(.type);
+multi method box(anyNode:D $node, *%c) {
+    $node.Reference;
+    my $config = %c<config> // self.config;
+    my $class := $config.class-from($node.type);
     $class.REPR.starts-with('C')
```

Comment:
Can this now be simplified by removing `$class.REPR.starts-with('C')
        ?? nativecast($class, $node)
        !!`

Reply:
BTW, probably yes. I was considering this move at some point, but there were some `CPointer` box-classes still lying around at the moment, so it slipped of my mind.

It's not the only thing that could be done here. I'm committing the change right now.